tutoraspects/templates/aspects/apps/superset/pythonpath/openedx_sso_security_manager.py

```diff
@@ -59,7 +61,9 @@ def oauth_user_info(self, provider, response=None):
         if provider == "openedxsso":
             user_profile = self.decoded_user_info()
 
-            user_roles = self._get_user_roles(user_profile.get("preferred_username"))
+            language_preference = request.cookies.get('openedx-language-preference', 'en')
+
+            user_roles = self._get_user_roles(user_profile.get("preferred_username"), language_preference)
```

Comment:
I think you need to filter the language to our supported languages here?

Reply:
I didn't know that, I just wanted to explore this idea but it's not a priority right now.

Yes, I should filter it